### PR TITLE
Frontend UI display

### PR DESCRIPTION
**Pull request by KartikPat250905**
Added card building function, with hovering effect on tokens.

### CODE REVIEW

File: src/frontend/index.html [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
-    </style>
-    <script type="module" src="/src/main.js"></script>
-  </head>
-  <body>
-    <div id="app"></div>
-  </body>
+      *{
+        margin: 0;
+        padding: 0;
+      }
```
Comment (unaisansegundo) [inline src/frontend/index.html]: We should have this CSS code in the index.css with the rest

**Review by ktnlvr — APPROVED** [review]

File: src/frontend/src/main.js [hunk 2]
```diff
@@ -1,21 +1,115 @@
...
+}
+
+function tooltip(tokenType) {
+  if (tokenType === "square")
+    return `This is a trap.`
+  else
+    return `This is nothing.`
+}
+
+function createCard(parent, charId, imgSrc, imgAlt, tokenTypes = []) {
```
Comment (unaisansegundo) [inline src/frontend/src/main.js:68]: Does this function need to define each element of the whole character and card structure one by one every time we create any of them or can we make it so each main component inside is defined once and reused.

File: src/frontend/src/main.js [hunk 3]
```diff
@@ -1,21 +1,115 @@
...
+    cardCounts[charId] = {};
+  }
+  tokenTypes.forEach((_, index) => {
+    cardCounts[charId][index + 1] = 0;
+  });
+}
+
+function updateTokenCount(token) {
+  //This is the token clicking hook here you can change and get info of the selected token ,inc or dec the count corresponding count to it.
+  //Remember to update the cardCounts data structure accordingly for smoother data storing.
```
Comment (unaisansegundo) [inline src/frontend/src/main.js:50]: I'm not sure I see the purpose of this comment

File: src/frontend/src/main.js
Comment (unaisansegundo) [inline src/frontend/src/main.js:1]: I think we should store this type of drawing logic out of the main file and into their own components.

**Review by unaisansegundo — COMMENTED** [review]
Implement changes

**Pull request merged**